capa/main.py

```diff
@@ -390,10 +392,13 @@ def is_supported_format(sample: Path) -> bool:
     return len(list(capa.features.extractors.common.extract_format(taste))) == 1
 
 
-def is_supported_arch(sample: Path) -> bool:
+def is_supported_arch(sample: Path) -> Union[str, int, float, bytes]:
```

Comment:
I think this should still return `bool`. `get_arch` can return the string/handle the cases.

Reply:
I will change that to use get_arch instead